crates/iota/src/name_commands.rs

```diff
@@ -544,6 +558,34 @@ impl NameCommand {
                     .await?,
                 )
             }
+            Self::UnsetTargetAddress { domain, opts } => {
+                let entry = get_registry_entry(&domain, &iota_client).await?;
+                if entry.name_record.target_address.is_none() {
+                    anyhow::bail!("target address is already unset");
+                }
+
+                let nft = get_proxy_nft_by_name(&domain, context).await?;
+                let iota_names_config = get_iota_names_config(&iota_client).await?;
+
+                NameCommandResult::Client(
+                    IotaClientCommands::Call {
+                        package: nft.package_id(&iota_client).await?,
+                        module: nft.module_name().to_owned(),
```

Comment:
This is `subdomains` or `subdomain_proxy`, but it should be `controller` or `subdomain_proxy` here`?

Reply:
I changed the `module_name` fn

Reply:
I don't see any change to `module_name`?

Reply:
Where did you change it? Did you not push something maybe?

Reply:
Pushed 😅